react/Chip/Readme.md

```diff
@@ -46,3 +46,76 @@ const ContactChip = ({ contact }) => (
   <Chip.Button disabled><Icon icon='back' /></Chip.Button>
 </div>
 ```
+
+### Specify a root component
```

Comment:
Specify underlying tag/component ?

This was the only example that at first glance was not completely clear to me.